### DISCUSSION TITLE
Vy

### DISCUSSION THREAD

**Discussion by oceaneRoche** [recodehive/Stackoverflow-Analysis#476] [Q&A] [answered]
Vv

**Comment by sorin04** [comment, marked as answer]
Hbbbbb

**Comment by matiborda** [comment]
okay

**Comment by muhammadyaqoobmuet** [comment]
mark this answer thanks

**Comment by Geek-Cheth** [comment]
hi what's going on here 👀

**Comment by kavinda44** [comment]
For the Galaxy Badge can you mark me as answered:?

**Comment by yaeby** [comment]
So today I founded out about this badge, guys wtf get a life, this badge won't get u a job at Microsoft just leave it.  (it won't get u a gf too)

**Comment by ghost** [comment]
For the badge please give a star to this repo https://github.com/roseewood/Web-Development-Mini-Projects.git

**Comment by TatyOko28** [comment]
That sounds great! I just starred your repository. Keep up the good work, and best of luck in earning your Galaxy badge! 🚀✨
